src/features/SearchErrorPage/styled.js

```diff
@@ -0,0 +1,64 @@
+import styled, { css } from "styled-components";
+
+export const ErrorContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 180px;
+  padding: 0px 20px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.large}) {
+    margin-top: 40px;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.medium}) {
+    padding: 5px;
+  }
+`;
+
+export const ErrorImage = styled.img`
+  @media (max-width: ${({ theme }) => theme.breakpoints.medium}) {
+    width: 80px;
+  }
+`;
+
+export const ErrorMessage = styled.p`
+  font-size: 36px;
+  font-weight: 600;
+  font-style: normal;
+  margin: 24px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.medium}) {
+    font-size: 28px;
+  }
+
+  ${({ smaller }) =>
+    smaller &&
+    css`
+      font-size: 22px;
+      font-weight: 500;
+      text-align: center;
```

Comment:
It should go to main styles of ErrorMessage, because on smaller screens the main text aligns to left.

Reply:
I wrote about the last line (text-align), so you did what I meant 😉